model/flow/execution_receipt.go

```diff
@@ -28,6 +30,7 @@ func ExecutionReceiptFromMeta(meta ExecutionReceiptMeta, result ExecutionResult)
 type ExecutionReceipt struct {
 	ExecutorID        Identifier
 	ExecutionResult   ExecutionResult
+	ResultSignature   crypto.Signature // signature only over the ExecutionResultID; for aggregation in Seal
 	Spocks            []crypto.Signature
 	ExecutorSignature crypto.Signature
```

Comment:
Could we comment on the different use cases for `ResultSignature` and `ExecutorSignature`?

They both are signed on the hash of some fields, and then added back to the `ExecutionReceipt` data, which could potentially change the hash of those fields that were signed. In that happen, it could make the signature invalid. We probably need to have a test case for it.

Reply:
removed `ExecutionReceipt.ResultSignature` entirely